### PR TITLE
Replaced all instances of armour/armoury with armor/armory

### PR DESCRIPTION
**Pull request by mikomyazaki**
Armor code was a bit of a mess due to inconsistent armor/armour all over the place. 

This shouldn't affect anything in the cargo database.

### CODE REVIEW

**Comment by mikomyazaki** [comment]
!review

File: code/modules/clothing/spacesuits/rig/suits/ert.dm [hunk 1]
```diff
@@ -98,7 +98,7 @@
 
 /obj/item/rig/ert/assetprotection
 	name = "heavy asset protection suit control module"
-	desc = "A heavy suit worn by the highest level of Asset Protection, don't mess with the person wearing this. Armoured and space ready."
+	desc = "A heavy suit worn by the highest level of Asset Protection, don't mess with the person wearing this. armored and space ready."
```
Comment (alsoandanswer) [inline code/modules/clothing/spacesuits/rig/suits/ert.dm:101]:
```suggestion
	desc = "A heavy suit worn by the highest level of Asset Protection, don't mess with the person wearing this. Armored and space ready."
```

should probably fix the other armors that start with caps at the front

Comment (mikomyazaki) [reply code/modules/clothing/spacesuits/rig/suits/ert.dm]: Done, thanks.

File: code/modules/mob/living/carbon/human/species/species_attack.dm [hunk 2]
```diff
@@ -176,7 +176,7 @@
 /datum/unarmed_attack/bite/infectious
 	shredding = 1
 
-/datum/unarmed_attack/bite/infectious/apply_effects(var/mob/living/carbon/human/user,var/mob/living/carbon/human/target,var/armour,var/attack_damage,var/zone)
+/datum/unarmed_attack/bite/infectious/apply_effects(var/mob/living/carbon/human/user,var/mob/living/carbon/human/target,var/armor,var/attack_damage,var/zone)
```
Comment (alsoandanswer) [inline code/modules/mob/living/carbon/human/species/species_attack.dm:179]: duplicate variable in this file at line 187

**Pull request merged**